Test/Test.csproj

```
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net48</TargetFrameworks>
+    <TargetFrameworks>net8.0;net7.0;net6.0;net48</TargetFrameworks>
```

```suggestion
    <TargetFrameworks>net8.0;net6.0;net48</TargetFrameworks>
```
I think we don't need tests for net7.0, will be out of support soon.